### DISCUSSION TITLE
AI對社會組的影響

### DISCUSSION THREAD

**Discussion by xipeiguo** [1101-GS4719-C/Python-Community-GS4719-C#2043] [General]
https://opinion.cw.com.tw/blog/profile/441/article/14992
當今的學生無論是選擇商管類科系或是文史哲相關領域都有不少同學試著去學習AI的應用，我認為有一部份原因是個人興趣，然而多數應該還是社會中無形的壓力推著我們去學習，以增加自身的競爭力，有些同學也因為社會組的科系薪水相對較低而選擇投入科技相關領域。然而是否在未來的某一天當科技發展相當成熟且能帶給人們更多便利之時，學生能更加尊崇內心的想法，選擇自己真正感興趣的科系，而對於社會對AI領域的需求則完全可以透過專業人士跟電腦自行的精進，以藉此達成專業分工的美好願景，我想這是我非常期待能看到的樣貌。

**Comment by Clare0808** [comment]
個人覺得現今社會的發展真的對社會組相當不利，大 多數的高薪工作幾乎都和自然組的科系相關，最典型的例子就是工程師。而在申請大學的部分，很多科系明明就是屬於社會組的，但有些比較好的學校卻會參採自然科以及數A，這對社會組的學生相當不公平。

**Comment by JoannaLai909** [comment]
我身邊有很多讀文組朋友，但他們卻有寫程式的能力，這讓我非常震驚，而且甚至寫得比我這個二類組的還要好。我問了他們原因，他們說因為怕自己以後的專業可能會被科技取代，所以想增進自己符合未來趨勢的能力

**Comment by ds0w0** [comment]
當前學生選擇學習AI技術確實有很大一部分來自社會的無形壓力，但我們也不能忽視AI技術與人文社科領域整合的潛力。透過跨領域的學習，不僅能增強個人的競爭力，更能為社會帶來新的價值與深度分析的能力。此外，我們也應重新思考成功的定義，不僅限於薪水和技術性工作，而是考慮對社會文化的長遠影響。我認為臺灣的教育也將會轉型，更加強化批判性思維和創意表達等軟技能的培養，以應對未來社會的多元需求。我相信，當社會逐漸達到這種整合與平衡的狀態時，學生們將能更自由地追求自己真正的興趣，而不再只是受到市場壓力的驅使。

**Comment by Nana1218** [comment]
我自己在這方面也是因為社會的趨勢所以被半強迫的選擇了學習AI，因為在台灣，自然組擁有絕大部分的薪資優勢，而身為社會組的我，想不被趨勢所淘汰的話，學習AI就變成了我的必備條件。

**Comment by hunteryang0108** [comment]
雖然AI的存在能協助我們持續發展社會組的興趣，但它也有可能取代社會組的某些工作。因此我認為如果想朝社會處持續發展，我們應該在這項興趣之外，發展至少一項能作為謀生工具的能力，成為所謂的Pi型人才。
